CHANGELOG.rst

```diff
@@ -4,6 +4,87 @@
 Synapse Changelog
 *****************
 
+v2.143.0 - TBD
+==============
+
+Model Changes
+-------------
+- Update to the ``crypto`` model.
+  (`#3256 <https://github.com/vertexproject/synapse/pull/3256>`_)
+
+  Updated Types
+  -------------
+
+  ``hex``
+    The ``zeropad`` option has been changed from a ``bool`` to an ``int``.
+    It now indicates how many bytes to left pad a value with zeros (``0x00``).
+
+  Updated Properties
+  ------------------
+
+  ``cyrpto:509:cert``
+    The form had the following properties updated on it:
+
+    ``serial``
+      The ``size`` value has been changed to ``zeropad`` to allow recording
+      certificates with serial numbers with less than 40 bytes.
```

Comment:
This also allows the use of larger than 40 byte hex values to account for malformed certs